package.json

```diff
@@ -18,8 +22,8 @@
   ],
   "scripts": {
     "watch": "tsc-watch --project=tsconfig.json",
-    "prebuild": "rimraf lib",
-    "build": "tsc",
+    "prebuild": "rimraf lib-cjs lib-esm",
+    "build": "tsc --target esnext --outDir lib-esm && tsc --target es5 --outDir lib-cjs",
```

Comment:
```suggestion
    "build": "tsc --target=esnext --outDir=lib-esm && tsc --target=es5 --outDir=lib-cjs",
```

Reply:
If I am not wrong, tsc doesn't work with `=`

Reply:
Ah yeah. Shame.